static/js/src/contributions.js

```diff
@@ -4,77 +4,77 @@
 function chooseEquivalentItem(totalAmount) {
   const items = [
     {
-      imageFile: "86981cf1-skull.png?w=78",
+      imageFile: "76474def-skull.png?w=78",
       price: 0,
       description: "Nothing. Use Ubuntu for free.",
```

Comment:
Since the full sentence is "The same price as X", the "nothing" here should be lower case
```suggestion
      description: "nothing. Use Ubuntu for free.",
```